src/pipeline/format.js

```diff
@@ -43,62 +66,53 @@ export default function formatEvent(attributes = {}) {
     classification,
     created,
     lastModified,
-    calName,
     htmlContent
   } = attributes
 
   let icsFormat = ''
-  icsFormat += 'BEGIN:VCALENDAR\r\n'
-  icsFormat += 'VERSION:2.0\r\n'
-  icsFormat += 'CALSCALE:GREGORIAN\r\n'
-  icsFormat += foldLine(`PRODID:${productId}`) + '\r\n'
-  icsFormat += foldLine(`METHOD:${method}`) + '\r\n'
-  icsFormat += calName ? (foldLine(`X-WR-CALNAME:${calName}`) + '\r\n') : ''
-  icsFormat += `X-PUBLISHED-TTL:PT1H\r\n`
   icsFormat += 'BEGIN:VEVENT\r\n'
-  icsFormat += `UID:${uid}\r\n`
-  icsFormat +=  foldLine(`SUMMARY:${title ? setSummary(title) : title}`) + '\r\n'
```

Comment:
I think this was a bug and the intent was to not include the line if there was no title?